site/app/templates/grading/electronic/Details.twig

```diff
@@ -764,25 +766,14 @@
     }
 
     function change_inquiry() {
-        var button = document.getElementById('inquiryButton');
-        var status = Cookies.get('inquiry_status');
-        if(status === 'on') {
-            status = 'off'
-        }
-        else{
-            status = 'on';
-        }
-        if(status === 'on') {
-            Cookies.set('inquiry_status', 'on');
-            var inquiry = Cookies.get('inquiry_status');
-            grade_inquiry_only();
-            button.textContent = "Grade Inquiry Only: On";
-        } else {
+        const status = {{ inquiry_status ? 1 : 0}};
+        if (status) {
             Cookies.set('inquiry_status', 'off');
-            var inquiry = Cookies.get('inquiry_status');
-            reverse_inquiry_only();
-            button.textContent = "Grade Inquiry Only: Off";
+        } 
+        else {
+            Cookies.set('inquiry_status', 'on');
         }
```

Comment:
The status constant isn't needed here.  Can't you just use Twig to conditionally set the value of the cookie?

Reply:
It was rather consistent across the many change functions. Would you like me to refactor all of them?